docs/resources/service.md

```diff
@@ -19,10 +35,14 @@ description: |-
 
 ### Optional
 
-- **description** (String, Optional)
-- **id** (String, Optional) The ID of this resource.
-- **service_tier** (Integer, Optional) The Service Tier of this resource - between 1 - 5.
-- **labels** (Map of String, Optional)
-- - **add_on_alert** (Boolean, Optional)
+- **add_on_alert** (Boolean, Optional) Indicates if FireHydrant should automatically create 
+   an alert based on the integrations set up for this service, if this service is added to an 
+   active incident. Defaults to `false`.
+- **description** (String, Optional) A description for the service.
+- **labels** (Map of String, Optional) The name of the service.
```

Comment:
should this be `**name**`? The description makes me think yes?

Reply:
And if `**labels**` is needed, maybe that needs to be added with a different description?

Reply:
i think this was meant to be "Labels" because Name is not optional and labels are? If i understand the sections correctly